src/core/memory.cc

```diff
@@ -176,6 +176,13 @@ struct hash<seastar::allocation_site> {
 
 }
 
+#if FMT_VERSION >= 90000
+namespace seastar::memory {
+struct human_readable_value;
+}
```

Comment:
Why not put it after human_readable_value is declared?

Reply:
just wanted to minimize the size of header files if possible. so unless a type is very popular i'd prefer keep the specialization in the place where it is used. 

but anyway, will put it close to where the type is declared as you're proposing.